streams/src/test/java/org/apache/kafka/streams/integration/QueryableStateIntegrationTest.java

```diff
@@ -432,7 +452,8 @@ public void queryOnRebalance() throws Exception {
                     listeners.get(i),
                     inputValuesKeys,
                     storeName + "-" + streamThree,
-                    DEFAULT_TIMEOUT_MS);
+                    DEFAULT_TIMEOUT_MS,
```

Comment:
should we also move over the standby querying test from the original PR? I did not see any other test covering querying with standby replication.. @vvcephei ?

Reply:
Thanks for pointing this out, @vinothchandar .

The ability to query standbys (and hence this test) is part of the follow-up PR: https://github.com/apache/kafka/pull/7962